docs/.eleventy/layouts/main.njk

```diff
@@ -5,25 +5,30 @@ description: "Snowpack is the fast, bundle-free build tool for all kinds of fron
 ---
 
 <div class="grid-body-header">
-  <svg height="64px" style="padding-left: 8px;" viewBox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-      <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
-          <g id="mountain-solid" transform="translate(-1.000000, 0.000000)"fill-rule="nonzero">
-              <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
-          </g>
+  <svg height="64px" style="padding-left: 8px;" viewbox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
+      <g id="mountain-solid" transform="translate(-1.000000, 0.000000)" fill-rule="nonzero">
+        <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
       </g>
+    </g>
   </svg>
   <h1 class="header-snowpack">Snowpack</h1>
   <p class="header-snowpack-subtitle">
     The faster frontend build tool.
   </p>
+  <p class="header-snowpack-blurb">
+    Snowpack is an unbundled front-end development environment that transforms your code into JavaScript's module system (<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import">ESM</a>) within 50 milliseconds. See your code run in the browser as you code —which means no more waiting for a bundler. When it's time to deploy, Snowpack works with your favorite tools (including bundlers) to create a fast, production-ready build. 
```

Comment:
This is a great step forward in concise-ness, perfect length without overloading.

On the flip side, the second sentence is a step backwards in terms of clarity. The tricky part of this for me is that every bundler / existing tool also claims to be fast and easy to use, so just saying that we're fast and easy to use never really buys us anything (vs. all the other amazing stuff that we can talk about).

Here's how I edited this for the Storybook team when I sent them a blurb:

> Snowpack is an unbundled front-end development environment that leverages JavaScript's native module system (ESM) to start up & react to changes in 50 milliseconds or less. When it's time to deploy, Snowpack works with your favorite tools (including bundlers) to create a fast, production-ready build.

#### But that's still not perfect. Here's where I landed on a second attempt with a bit more time (splitting up the first sentence into two seemed to help):

> Snowpack is an unbundled front-end build tool. It's fast... *really fast.* By leveraging JavaScript's native module system (<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import">ESM</a>) Snowpack can start up in ~50 milliseconds and reflect changes in the browser instantly. When it's time to deploy, Snowpack works with your favorite tools and bundlers to help you create a fast, production-ready build.

^ what to do you think?

Reply:
I'm unsure about the `unbundled` in the first sentence. It makes it sound as if Snowpack itself is unbundled. Maybe remove it from the beginning and use it next to "leveraging JavaScript's native module system" as a means to explain how Snowpack achieves it's fast start up and processing time?

Reply:
maybe "modern" instead of "unbundled"?

Also in re-reading this, I'm less happy with starting a sentence with "By blah blah blah". Another ground-up rewrite:

> Snowpack is a modern front-end build tool that replaces traditional bundlers like Webpack or Parcel. Snowpack starts up in less than 50 milliseconds and rebuilds changed files instantly. Connect your favorite build tools and plugins to customize your build and create a fast, production-ready site.

@melissamcewen I think at this point I'm just spinning my wheels, would love to hear your thoughts!